pkg/proxy/proxy_community_test.go

```diff
@@ -287,73 +287,103 @@ func (s *TestProxySuite) checkProxyCommunityOK(fakeApp *fake.ProxyFakeApp, p *Pr
 			},
 			// Given user alice exists
 			// And   alice owns a private workspace
-			// When  alice requests the list of pods in a non existing namespace in alice's workspace
-			// Then  the proxy does NOT forward the request
-			// And   the proxy rejects the call with 403 Forbidden
-			"plain http request as owner to not existing namespace in private workspace": {
-				ProxyRequestMethod:          "GET",
-				ProxyRequestHeaders:         map[string][]string{"Authorization": {"Bearer " + s.token(alice)}},
-				ExpectedProxyResponseStatus: http.StatusForbidden,
-				RequestPath:                 podsInNamespaceRequestURL("alice-private", "not-existing"),
-				ExpectedResponse:            "invalid workspace request: access to namespace 'not-existing' in workspace 'alice-private' is forbidden",
+			// When  alice requests the list of pods in a namespace which does not belong to the alice's workspace
+			// Then  the proxy does forward the request anyway.
+			// It's not up to the proxy to check permissions on the specific namespace.
+			// The target API server will reject the request if the user does not have permissions to access the namespace.
+			// Here the request is successful because the underlying mock target cluster API server returns OK
```

Comment:
```suggestion
			// Here the request is successful because the underlying mock target cluster API always server returns OK
```

Just to make it clear that we are referring to the way mock target cluster is setup and not this particular request?